pkg/web/apps/apps_service.go

```diff
@@ -34,3 +35,17 @@ func (a *appsService) GetApps(c echo.Context) (*[]models.App, error) {
 
 	return apps, nil
 }
+
+// GetAppsId retrieves app by id from the repository
+func (a *appsService) GetAppsID(c echo.Context) (*[]models.App, error) {
+
+	apps, err := a.psqlRepository.GetAppsID(c)
+	if err == nil && apps == nil {
+		return nil, httperrors.NotFound(c, "App not found")
```

Comment:
What HTTP code will it return? Because the request in this situation was made with success the data just do not exist.

Reply:
see swagger docs chrome-extension://ljlmonadebogfjabhkppkoohjkjclfai/index.html?url=https://github.com/aerogear/mobile-security-service/pull/17/files#!/apps/getAppById